develop-docs/self-hosted/reverse-proxy.mdx

```diff
@@ -42,53 +42,7 @@ Endpoint for health checks is available on `/_health/` endpoint using HTTP proto
 
 We recommend installing NGINX since that's what we are using on [sentry.io](https://sentry.io/).
```

Comment:
Actually uhh, just became aware we provide nginx in the default install, so probably have to change this wording too.

Reply:
The configuration I removed is for an nginx deployment outside of the self-hosted docker compose environment, should probably keep that...